File: 20170116/20170116_FirstTest/FirstTest170116/ViewController.m
//
//  ViewController.m
//  FirstTest170116
//
//  Created by ji jun young on 2017. 1. 16..
//  Copyright © 2017년 Fastcampus. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    NSLog(@"브레이크 포인트 전") ;
    
    NSLog(@"브레이크 포인트 후") ;
    
    NSLog(@"중간 공백....") ;
    
    NSLog(@"다음 브레이크 포인트 전") ;
    
    NSLog(@"다음 브레이크 포인트 후") ;
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)btnAction
{
    
    [self.titleLB setText:@"텍스트바뀜\n텍스트가 잘 바뀌나"];
    
    NSLog(@"버튼이 활성화 되었음") ;
}


- (IBAction)swtAction:(UISwitch *)sender
{
    
    if (sender.isOn == YES) {
        [self.titleLB setText:@"TEST...텍스트 추가\n 작성 얼마나 길게 써지는지..."];
        NSLog(@"스위치가 켜졌음") ;
    }else{
        [self.titleLB setText:@"텍스트바뀜\n텍스트가 잘 바뀌나"];
        NSLog(@"스위치가 꺼졌음") ;
    }

    
    
}



@end

File: 20170510/stringAndNumbers/stringAndNumbers/ViewController.swift
//
//  ViewController.swift
//  stringAndNumbers
//
//  Created by ji jun young on 2017. 6. 13..
//  Copyright © 2017년 Fastcampus. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        let distance:Float = 69500
        
        let divider:Float = 1000
        
        let script:String = "판교에서 파주까지의 거리는" + String(distance / divider) + "입니다."
        
        let script2:String = "판교에서 파주까지의 거리는 \(distance / divider) 입니다."
        
        print(script)
        
        print(String(distance / divider))
        
        //튜플의 사용
        
        let time1 = (9, 0, 48.2)
        
        let time2:(h:Int, m:Int, s:Float) = (11, 7, 52.4)
        
        //인티저, 플롯 등 여러 값을 섞을 수 있다
        
        let duration = (time1, time2)
        
        let (start, end) = duration
        
        let endHour = duration.1.h
        
        let endHour2 = time2.h
        
        let endHour3 = end.h
        
        typealias Time = (h:Int, m:Int, s:Float)
        typealias duration2 = (start:Time, end:Time)
        
        let today:duration2 = ((9, 5, 25.4), (11, 4, 45.3))
        
        print("우리는 \(today.end.h)시까지 공부했다.")

        
        
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

